### ISSUE TITLE
API Particle Switch Hook using name - effect does not switch off if other particle effects are also currently on (Foundry V10)

### ISSUE THREAD

**Issue by napolitanod** [ghost-fvtt/fxmaster#320]
### Expected Behavior

Reading through the API documentation, hook call "fxmaster.switchParticleEffect" should toggle an effect off and on and can target based on effect id (name).

### Current Behavior

 The hook call DOES behave this way (toggles the effect off when called), but only if no other particle effect is also currently running (for the instances that I have run tests for, see below). If another effect is running then the targeted effect will not switch off.

### Steps to Reproduce

execute the following:

```js

const wait = (delay) => new Promise((resolve) => setTimeout(resolve, delay));

Hooks.call("fxmaster.switchParticleEffect", {
    name: "Man2LrOJLHbaIRnK",
    type: "snowstorm",
    options: {
       "density": 0.6,
       "direction": 55,
       "lifetime": 1.9,
       "scale": 0.4,
       "speed": 2.9,
       "tint": "#12f3ef"
       }
});

await wait(3000);

Hooks.call("fxmaster.switchParticleEffect", {
            name: "Man2LrOJLHbaIRnK",
            type: "snowstorm"
          });
```

If it behaves as my session behaves then the snowstorm effect will come on, last for 3 seconds, and then switch off

now execute the following:

```js
const wait = (delay) => new Promise((resolve) => setTimeout(resolve, delay));

Hooks.call("fxmaster.switchParticleEffect", {
    name: "LvsafasfHbaIRnK",
    type: "snowstorm",
    options: {
       "density": 0.6,
       "direction": 55,
       "lifetime": 1.9,
       "scale": 0.4,
       "speed": 2.9,
       "tint": "#12f3ef"
       }
});

await wait(3000);

Hooks.call("fxmaster.switchParticleEffect", {
    name: "Man2LrOJLHbaIRnK",
    type: "snowstorm",
    options: {
       "density": 0.6,
       "direction": 55,
       "lifetime": 1.9,
       "scale": 0.4,
       "speed": 2.9,
       "tint": "#12f3ef"
       }
});

await wait(3000);

Hooks.call("fxmaster.switchParticleEffect", {
            name: "Man2LrOJLHbaIRnK",
            type: "snowstorm"
          });
```

In my session, the first snowstorm starts. then, 3 seconds later, the second part is run and more snow particles are added. Then, 3 seconds later, the switch to turn off the snowstorm with name "Man2LrOJLHbaIRnK" is executed. I would expect either all of the snowstorm to end at that point or just the portion that was added when the second part was run. However, no snowstorm particles are switched off.

For ease, my example uses the same effect. But this can be replicated with another particle effect running. For example, if birds are flying on my screen and I run the first test above, which creates the snowstorm once and then runs the hook to switch it off 3 seconds later, the snowstorm will not switch off. 

### Context

I also tested with passing in the options with the switch 'off' call for name "Man2LrOJLHbaIRnK" but that did not work.

let me know if you need anything more!

thanks

Dan

### Version

v3.1.3

### Foundry VTT Version

v10 testing 1 build 275

### Operating System

Windows

### Browser / App

Native Electron App

### Game System

dnd5e (2.0.0 - alpha 2)

**Comment by ghost91-** [comment]
Thanks for the report, I’ll try to take a look this evening.

**Comment by ghost91-** [comment]
@napolitanod I just released version 3.1.4 to address this issue. Can you verify that it works correctly now?

**Comment by napolitanod** [comment]
Ya, works great now! thank!!!!!
